build/toolchain/win/msvc_toolchain.gni

```diff
@@ -16,6 +16,7 @@ import("//build/toolchain/cc_wrapper.gni")
 import("//build/toolchain/toolchain.gni")
 
 tool_wrapper_path = rebase_path("tool_wrapper.py", root_build_dir)
+assert(use_cobalt_customizations)
```

Comment:
Add a helpful message here and let's run it immediately (before even imports occur)

Reply:
added before imports. 
I think it's best to let the developer see the definition use_cobalt_customizations : as that can change over time.